source/manual/setting-up-new-rails-app.html.md

```diff
@@ -21,7 +21,7 @@ For most apps, this means you'll be using:
 - [Jenkins](https://jenkins.io) for CI
```

Comment:
at the section 'Bootstrapping a new application' there is another mention of errbit